src/dom/dom.js

```diff
@@ -384,46 +384,44 @@ tags.forEach(function(tag) {
 });
 
 /**
- * Creates an &lt;img&gt; element in the DOM with given src and
- * alternate text.
+ * Creates an &lt;img&gt; element in the DOM with given source
  *
  * @method createImg
  * @param  {String} src src path or url for image
- * @param  {String} alt <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Attributes">alternate text</a> to be used if image does not load. You can use also an empty string (`""`) if that an image is not intended to be viewed.
```

Comment:
I think we'd like to keep `alt` as a required argument for the shorter version of this, to encourage people to use alt text for their images